Source/JavaScriptCore/jit/SimpleRegisterAllocator.h

```diff
@@ -217,56 +208,32 @@ class SimpleRegisterAllocator {
         Register findMin()
```

Comment:
nit: Can we rename `findMin` to `findRegisterToSpill` or something more descriptive of what it does?

Reply:
Can you also ASSERT at the top of this function that `m_freeRegisters` is empty?